templates/zerver/help/import-data-from-slack.md

```diff
@@ -53,3 +53,5 @@ If the users are not from the same organization, you should change your organiza
 - Zulip's "default streams" work slightly differently from Slack's
   "Default channels" -- new users are automatically subscribed, but
   users can still unsusbcribe from them.
+- As user phone and skype data are not stored in Zulip, they are not preserved in
```

Comment:
@rht Should the wordings be changed here?

Reply:
Would be clearer with "phone number" and "Skype username" rather than "data." Also, should be "they are not preserved during the conversion."